apiserver/facades/schema.json

```diff
@@ -32959,7 +32963,7 @@
                         "Patch": {
                             "type": "integer"
                         },
-                        "Series": {
+                        "Release": {
```

Comment:
So this is worrying, either it's a Find&Replace issue or it's caused a schema change. If it's a schema change then we should seriously work out why.

Reply:
I have a TODO in the PR description:
"serialisation of version.Binary in a few API calls needs a compatibility shim"

Sadly we use version.Binary directly in apiserver/params (with no tags in the struct). I need to analyse the use cases and see what needs a compatibility shim added. I didn't want to make this PR any bigger bu doing that work here.